docs/appendices/release-notes/unreleased.rst

```diff
@@ -64,6 +64,10 @@ Changes
 - Added support for ``ORDER BY``, ``MAX``, ``MIN`` and comparison operators on
   expressions of type ``INTERVAL``.
 
+- Added a back-off policy to ``COPY FROM`` retry logic to pause temporarily
+  in exponential increments between retries caused by ``SocketExceptions`` and
+  ``SocketTimeoutExceptions``.
```

Comment:
I'd phrase it more user facing than implementation wise:
```suggestion
- Improved ``COPY FROM`` retry logic to retry exponentially on temporary
  network timeout and general network errors.
```

Reply:
Thank you! updated.